book/src/language-features/s-strings.md

```diff
@@ -36,10 +36,13 @@ join s=salaries side:left [
 ]
 ```
 
-For those who have used Python, s-strings are similar to Python's f-strings, but
-the result is SQL code, rather than a string literal. For example, a Python
-f-string of `f"average{col}"` would produce `"average(salary)"`, with quotes;
-while in PRQL, `s"average{col}"` produces `average(salary)`, without quotes.
+For those who have used python, s-strings are similar to python's f-strings, but
```

Comment:
I have zero view here. This was recently changed to Title Case, which I merged, since it does appear in the docs as Python rather than python; even though the logo is lower case. But I hesitate to comment, please ignore if you have any actual view

Reply:
oh yeah, that's consequence of a bad merge, whoops